personal/privacyidea/PDF/class_RecoveryKeyPDF.inc

```diff
@@ -63,7 +63,7 @@
         $this->Cell(12, 10, _("Subject: Recovery key"));
         $this->Ln(12);
         $this->SetFont('DejaVu', '', 10);
-        $this->Cell(12, 10, sprintf(_("Dear %s %s,"), $givenname, $sn));
+        $this->WriteHTML(sprintf(_("Dear %s,"), $this->Salutation($sn, $givenname, $academictitle)));
```

Comment:
This makes assumptions based on English/German but is locale-specific, better to remove the "Dear" and have `Salutation()` return the whole line.

Reply:
Noted in internal notes.